damnit/gui/main_window.py

```diff
@@ -348,6 +348,17 @@ def autoconfigure(self, path: Path, proposal=None):
         self._tab_widget.setEnabled(True)
         self.show_default_status_message()
         self.context_dir_changed.emit(str(path))
+        self.launch_update_computed_vars()
+
+    def launch_update_computed_vars(self):
+        log.debug("Launching subprocess to read variables from context file")
+        proc = QtCore.QProcess(parent=self)
```

Comment:
As this is at the moment, `subprocess`  would be fine. But if we want to e.g. display a status message if this process exits with an error, using a QProcess makes that easy, because it will emit a [`finished` signal](https://doc.qt.io/qt-5/qprocess.html#finished). In GUI code, I try to use Qt versions of anything that has 'wait for stuff' APIs - like `Popen.wait()` or `.communicate()` - if that's straightforward.